WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1284,16 +1284,14 @@ - (UIAlertAction *)quickSaveAlertAction
             return [UIAlertAction actionWithTitle:NSLocalizedString(@"Publish Now", "Title of button allowing the user to immediately publish the post they are editing.")
                                             style:UIAlertActionStyleDestructive
                                           handler:^(UIAlertAction * _Nonnull action) {
-                                              self.post.status = PostStatusPublish;
-                                              self.post.dateCreated = [NSDate date];
+                                              [self.post publishImmediately];
                                               [self saveAction:action];
                                           }];
         } else {
             return [UIAlertAction actionWithTitle:NSLocalizedString(@"Submit for Review", "Title of button allowing a contributor to a site to submit the post they are editing for review.")
                                             style:UIAlertActionStyleDestructive
                                           handler:^(UIAlertAction * _Nonnull action) {
-                                              self.post.status = PostStatusPending;
```

Comment:
Well, the call to `publishImmediately` would then go and override that status with Publish. I could set it _afterwards_, I guess - or leave it to the server?